packages/common/typings/express.d.ts

```diff
@@ -30,7 +30,7 @@ declare global {
             originatedFromDashboard: boolean | undefined
             authenticationError: (message?: string) => unknown
             forbiddenError: (message?: string) => unknown
-            validationError: (message?: string) => unknown
+            validationError: (message?: string, errors?: any) => unknown
```

Comment:
Please remove this and use `userInputError` instead, so we are consistent with the graphql api.